README.md

```diff
@@ -73,6 +73,7 @@ with the following options:
 - List of extensions: `*.twig`
 - Invocation:
     - Parser command: `<project>/vendor/bin/twig-gettext-extractor --sort-output --force-po -o %o %C %K -L PHP --files %F`
+        - (Windows user having problems? Try to use this instead `C:\\Users\path\to\your\project\vendor\bin\twig-gettext-extractor.bat --sort-output --force-po -o %o %C %K -L PHP --files %F`)
```

Comment:
Yeah, you are right! IT can be confusing somehow. But please add the line specifying that the project tag will not be replaced by Poedit!

I have just added the bat extension.